recipes/veque/all/conanfile.py

```diff
@@ -0,0 +1,43 @@
+import os
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
```

Comment:
```suggestion
required_conan_version = ">=1.33.0"
```

Please add `required_conan_version` because this recipe uses `tools.get()` with `strip_root`.